packages/one-app-bundler/webpack/webpack.common.js

```diff
@@ -73,6 +73,11 @@ module.exports = {
           },
         }],
       },
+      !nodeEnvIsProduction && {
```

Comment:
can you add a test ensuring that this does not make it into production and a test that shows it is included

Reply:
added